ppr-ui/src/components/registration/AmendmentDescription.vue

```diff
@@ -28,7 +28,11 @@
     >
       <v-row class="pt-6">
         <v-col class="pa-4 pl-5 summary-text">
-          <p>
+          <p v-if="isSecurityActNotice">
```

Comment:
Future state to expand on which PPR reg(s) have this copy, pending design review i think. Potentially all Crown Charge as well?

Reply:
All reg types can have a partial discharge if they have more than 1 active debtor. I would be surprised if Kevin is okay with this rule for other registration types.

Reply:
Copy that, Sara had flagged this in designs with some potential future implementations. Will 100% defer to Kevin tho! For now, only SANs :)